drivers/TinyGSM/TinyGsmFifo.h

```diff
@@ -0,0 +1,153 @@
+#ifndef TinyGsmFifo_h
+#define TinyGsmFifo_h
+template <class T, unsigned N>
+class TinyGsmFifo
+{
+public:
+	// Suppress warning about uninitialized variables because initializing them in an init function
+	// allows the compiler to optimize away the variables in case the class is only instantiated but
+	// never used.
+	// cppcheck-suppress uninitMemberVar
+	TinyGsmFifo()
```

Comment:
Come on man, the suppress message is right there. Read it!

Reply:
It could also be that the suppression has to be on the line the warning is issued on, which is 11, not 10.
That is,
`TinyGsmFifo() // cppcheck-suppress uninitMemberVar`